frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/VerifyMfaCodeHandler.java

```diff
@@ -446,4 +470,18 @@ private AuditService.MetadataPair[] metadataPairsForEvent(
         return Stream.concat(basicMetadataPairs.stream(), additionalPairs.stream())
                 .toArray(AuditService.MetadataPair[]::new);
     }
+
+    private String getInternalCommonSubjectIdentifier(UserProfile userProfile) {
+        try {
+            Subject internalCommonSubjectIdentifier =
+                    ClientSubjectHelper.getSubjectWithSectorIdentifier(
```

Comment:
I don't think this is the correct method to use - think we need to use the same thing that we use to calculate the pairwise id in the CheckReauthUserHandler.

In general as well, a task for you when you check these audit events in splunk is to check that all reauth audit events for a certain journey have the same pairwise id.